cub/benchmarks/bench/scan/exclusive/base.cuh

```diff
@@ -101,7 +101,14 @@ try
 
   size_t tmp_size;
   dispatch_t::Dispatch(
-    nullptr, tmp_size, d_input, d_output, op_t{}, wrapped_init_t{T{}}, static_cast<int>(input.size()), 0 /* stream */);
+    nullptr,
+    tmp_size,
+    d_input,
+    d_output,
+    op_t{},
+    wrapped_init_t{T{}},
+    static_cast<offset_t>(input.size()),
```

Comment:
Here, we used to cast to `int` and then promote again to `offset_t` (argument type of `Dispatch`). This is wrong for sizes larger than `INT_MAX`.